alacritty_terminal/src/term/mod.rs

```diff
@@ -795,6 +798,7 @@ impl<T> Term<T> {
             title_stack: Vec::new(),
             selection: None,
             regex_search: None,
+            size: *size,
```

Comment:
If we need a copy anyways, just let the callee move the value by copy instead of reference and then change this to just `size,`.